spec/controllers/email_actions_controller_spec.rb

```diff
@@ -19,6 +19,24 @@
     end
   end
 
+  describe "follow_discussion" do
+    before do
+      @user = FactoryGirl.create(:user)
+      @group = FactoryGirl.create(:group)
+      @group.add_member!(@user)
+
+      @discussion = FactoryGirl.build(:discussion, group: @group)
+      DiscussionService.create(discussion: @discussion, actor: @user)
+      DiscussionReader.for(discussion: @discussion, user: @user).unfollow!
+    end
+
+    it 'follows the discussion' do
+      DiscussionReader.for(discussion: @discussion, user: @user).following?.should be false
```

Comment:
@gdpelican heads up that suggestion doesn't work. it generates 
```ruby
false.false?
```
and there's no `false?` defined for false.

```ruby
expect(DiscussionReader.for(discussion: @discussion, user: @user).following?).to eq false
``` 
works